android/res/color/preference_summary_text.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
hmm, I am confused as to which one. I see that some files are with GNU and some are with Apache. The readme states that it should be GNU, but your cleanup PR on my branch had the Apache.